test/js-native-api/common.c

```diff
@@ -0,0 +1,49 @@
+#include <js_native_api.h>
+#include "common.h"
+
+#include <stdio.h>
+
+void add_returned_status(napi_env env,
+                         const char* key,
+                         napi_value object,
+                         napi_status expected_status,
+                         napi_status actual_status) {
+
+  char p_napi_message[100] = "";
+  napi_value prop_value;
+
+  if (actual_status == expected_status) {
+    snprintf(p_napi_message, 99, "Invalid argument");
```

Comment:
```suggestion
    snprintf(p_napi_message, sizeof(p_napi_message), "Invalid argument");
```

(Side note: why the `p_`? Do you know that?)

Reply:
Hi @addaleax 

Thank you for your message. Usually I use p_<something> for a name that indicates a pointer. Could you please suggest a better name?

Thank you in advance,
    @octaviansoldea

Reply:
> Usually I use p_ for a name that indicates a pointer.

Okay, that makes sense, but we generally don’t to this elsewhere :slightly_smiling_face:. How about just `napi_message`, or if you want to be explicit about the type, `napi_message_string`?